better_bookmarks/js/background.js

```diff
@@ -16,4 +16,4 @@ chrome.contextMenus.create({
 // Context menu onClicked listener.
 chrome.contextMenus.onClicked.addListener(function(info, tab) {
     alert('first');
-});
```

Comment:
What did you change here?  Did you just add a newline?

Reply:
Uh I think so.

Reply:
Alright, I was just wondering, because all I see is a newline and there was a red circle with a slash, so I didn't know if that meant there was content other than the newline.